packages/integrations/image/src/endpoints/dev.ts

```diff
@@ -1,10 +1,10 @@
 import type { APIRoute } from 'astro';
 import { lookup } from 'mrmime';
-// @ts-ignore
-import loader from 'virtual:image-loader';
 import { loadImage } from '../utils.js';
 
 export const get: APIRoute = async ({ request }) => {
+	const loader = globalThis.astroImage.ssrLoader;
```

Comment:
Fix pt. 1 - `getImage()` in dev will now only build a `_image?` src for local images.  The endpoint can always use the `ssrLoader` global (currently defaults to the built-in sharp service)